zebra/interface.c

```diff
@@ -1074,6 +1074,10 @@ void if_up(struct interface *ifp)
 
 	if (zif->flags & ZIF_FLAG_EVPN_MH_UPLINK)
 		zebra_evpn_mh_uplink_oper_update(zif);
+
+	if (!zif->speed_update)
```

Comment:
don't need to test for an existing event here, outside the scheduling lock - it'll get checked inside the `add_timer` call.